Tests/LibJS/test-js.cpp

```diff
@@ -76,6 +77,16 @@ TESTJS_GLOBAL_FUNCTION(mark_as_garbage, markAsGarbage)
     return JS::js_undefined();
 }
 
+TESTJS_GLOBAL_FUNCTION(detach_array_buffer, detachArrayBuffer)
+{
+    auto array_buffer = vm.argument(0);
+    if (!array_buffer.is_object() || !is<JS::ArrayBuffer>(array_buffer.as_object()))
+        return vm.throw_completion<JS::TypeError>(global_object);
+
+    auto& array_buffer_object = static_cast<JS::ArrayBuffer&>(array_buffer.as_object());
+    return JS::detach_array_buffer(global_object, array_buffer_object, vm.argument(1));
```

Comment:
```suggestion
    auto& array_buffer = static_cast<JS::ArrayBuffer&>(array_buffer.as_object());
    return JS::detach_array_buffer(global_object, array_buffer, vm.argument(1));
```

Reply:
We already have a value named `array_buffer` in this scope

Reply:
(The `array_buffer` / `array_buffer_object` names being copied from the $262Object impl)

Reply:
oh, I see. Not super nice, but fine :upside_down_face: